Headers/DebugServer2/Target/Windows/Process.h

```diff
@@ -52,6 +52,16 @@ class Process : public ds2::Target::ProcessBase,
   ErrorCode writeMemory(Address const &address, void const *data, size_t length,
                         size_t *nwritten = nullptr) override;
 
+private:
+  ErrorCode makeMemoryWritable(Address const &address, size_t length,
+                               std::vector<MemoryRegionInfo> &modifiedRegions);
+  ErrorCode
+  restoreRegionsProtection(const std::vector<MemoryRegionInfo> &regions);
+  ErrorCode getMemoryRegionInfoInternal(Address const &address,
+                                        MemoryRegionInfo &info);
+  DWORD convertMemoryProtectionFromLLDB(uint32_t protection);
```

Comment:
the naming makes sense, the other conversions were done inside a bigger function